docs/spec_draft.md

```diff
@@ -638,7 +638,8 @@ function, with corner cases TBD. Numeric precision is implementation-defined.
 Performs element-wise max operation on tensors `lhs` and `rhs` and produces a
 `result` tensor. For floating-point element types, it implements the `maximum`
 operation from the IEEE-754 specification. For complex element type, it performs
-lexicographic comparison on the (real, imaginary) pairs.
+lexicographic comparison on the (real, imaginary) pairs using LLVM's `fcmp oeq`
+and `fcmp ogt` compare instructions.
```

Comment:
Calling out LLVM here looks too low-level. How about we use our trusty "with corner cases TBD" formulation here and below?